test/Jobs.js

```diff
@@ -127,8 +127,11 @@ var testRetrieveJob = {
   }
 };
 var app;
-before(function () {
+
+before(function(done) {
   app = require("../server/server");
+  console.log("Waiting for 5 seconds for boot tasks to finish: ",new Date());
+  setTimeout(done,5000);
```

Comment:
I assume line 133 & 134 only need if we want to run only this test file.

Reply:
Yes, exactly